starknet_py/net/client.py

```diff
@@ -261,3 +264,21 @@ async def deploy(
             raise Exception("Transaction not received")
 
         return res
+
+    async def get_class_hash_at(
+        self,
+        contract_address: CastableToHash,
+        block_hash: Optional[CastableToHash] = None,
+        block_number: Optional[BlockIdentifier] = None,
+    ) -> str:
```

Comment:
Shoudldn't we return an int?

Reply:
Gateway's get_class_hash_at returns string, we could convert it to int if that's better.